test/cuda.jl

```diff
@@ -1,5 +1,8 @@
 using CUDA
 
+# FIXME
+if VERSION < v"1.6-"
```

Comment:
I tried with CUDA and GPUCompiler master, but CUDA is still broken for me. Doesn't seem to be related to Zygote though.

Reply:
OK, I needed the master branch of LLVM.jl as well. Now it works